src/Traits/ValidationAwareTrait.php

```diff
@@ -17,22 +17,23 @@
  */
 trait ValidationAwareTrait
 {
-    protected function handleResponseException(ResponseInterface $response): void
+    protected function dispatchValidationException(InvalidStatusCodeException $exception): InvalidStatusCodeException|ValidationErrorsException
```

Comment:
This decides whether to change `InvalidStatusCodeException` into `ValidationErrorsException`